api/app/src/main/kotlin/packit/controllers/RoleController.kt

```diff
@@ -21,13 +22,35 @@ class RoleController(private val roleService: RoleService)
         return ResponseEntity.ok(mapOf("message" to "Role created"))
     }
 
-    @DeleteMapping("/{name}")
+    @DeleteMapping("/{roleName}")
     fun deleteRole(
-        @PathVariable name: String
+        @PathVariable roleName: String
     ): ResponseEntity<Map<String, String?>>
     {
-        roleService.deleteRole(name)
+        roleService.deleteRole(roleName)
 
         return ResponseEntity.ok(mapOf("message" to "Role deleted"))
     }
+
+    @PutMapping("/add-permissions/{roleName}")
```

Comment:
its not really a post or delete because we are not creating/deleting the role. We are updating the role object i.e its a update operation. technically should be patch but spring didn't like patch - said it doesn't accept them